src/test/kotlin/com/igorwojda/integer/addupto/challenge.kt

```diff
@@ -4,7 +4,7 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun addUpTo(n: Int): Int {
-    TODO("not implemented")
+    return n*(n+1)/2
```

Comment:
nitpick: this is indeed the most optimal solution. I'd also suggest looking into aggregate operations: https://kotlinlang.org/docs/collection-aggregate.html